### DISCUSSION TITLE
Error on Connections Panel extra Json field

### DISCUSSION THREAD

**Discussion by PhoenixChamb** [apache/airflow#29196] [Q&A]
Airflow Version 2.5.0

If you create a Http connection via Web UI and you set Extra configuration such as {'verify': 'False'} for not verifiying certs.

![image](https://user-images.githubusercontent.com/122443108/215131839-3e51deb2-fc66-47b6-a6c8-11425d96ae91.png)

Extra_args are going to be empty. 

I had to set it at HttpSensor level (extra_options).

HttpSensor(
                    task_id= f"*",
                    poke_interval=120,
                    timeout=60*60*24,
                    http_conn_id='*',
                    method='HEAD',
                    extra_options={'verify': False},
                    endpoint =f'****',
                    on_failure_callback=_http_sensor_failure_callback,
                )

 I believe that if I create a Http Connection and I set in 'extra' {'verify': 'False'}.

The connection would have that configuration that would be applied at the moment of connecting. However, the dictionary that you passed from property through the web page remains empty.

I also check it on WebServer, if you create an HttpConnection (https in my case and certs are needed) and you press 'test' button JSON extra config field are always empty.

**Comment by Taragolis** [comment]
> I believe that if I create a Http Connection and I set in 'extra' {'verify': 'False'}.

According to the [documentation](https://airflow.apache.org/docs/apache-airflow-providers-http/stable/connections/http.html) extra field in Connection propagated to the headers

**Reply by PhoenixChamb** [reply]
Yes, you are right. The problem if in the web page. If I set 'verify': 'False' on Extra as a JSON the properties doesn't propagate and they are always empty.

This values are always empty.

![image](https://user-images.githubusercontent.com/122443108/215428088-0d48c8c6-52dd-4acc-af4c-9d41b54f8860.png)

Here extra_options are empty.

![image](https://user-images.githubusercontent.com/122443108/215428258-d3203b87-4d81-44a3-a0e3-587619a44f61.png)

Then here, 'get' method always returns 'False'

![image](https://user-images.githubusercontent.com/122443108/215428845-caea9c94-2b40-4918-8ec0-b62ef32433f5.png)

I finally solve it as you said using 'verify' False in the HttpSensor, but If I wan't to check directly from the web page clicking this button.

![image](https://user-images.githubusercontent.com/122443108/215429189-13b774dc-446a-4847-9b81-3c4eb07858cb.png)

It doesn't work as I can't pass path to the certs or setting verify false as JSON extra params.

**Reply by Taragolis** [reply]
Well this options not propagated from Connection object, it is only work when you call `run_and_check` and pass this parameters as it HttpSensor does. The name of argument could mislead users it is a fact but  documentation of HTTP Connection Object is correct - Connection Extra set to the headers, it doesn't mention that this fields could use any other parameters.

However I would like to mention that this doesn't mean that this behaviour is correct for all cases.
Apache Airflow is an OSS and which developed/driven mostly by regular Airflow users.

If you feel confident and know how make better user experience you could create a PR, see [CONTRIBUTORS_QUICK_START](https://github.com/apache/airflow/blob/main/CONTRIBUTORS_QUICK_START.rst) first. If you want to create PR you don't need to create Issue.

If you don't feel confident with Airflow, Python and etc you could create Airflow feature request, and some one of could pick it up. however it could take unknown time, from couple hours to years.

In additional I wan't to warn ⚠️  HttpHook uses in a lot of different providers, so changes might be not straightforward because it could affect a lot of different providers, but it doesn't mean that this is a some kind of stopper for new feature of Http Provider
